assignments/callbacks.js

```diff
@@ -23,26 +23,58 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 
 function getLength(arr, cb) {
+  return cb(arr.length);
   // getLength passes the length of the array into the callback.
 }
 
+getLength(items, function(length) {
+  console.log(length)
+});
+
 function last(arr, cb) {
+  return cb(arr[arr.length-1]);
   // last passes the last item of the array into the callback.
 }
 
+last(items, function(ultimo) {
+  console.log(ultimo)
+});
+
 function sumNums(x, y, cb) {
+  return cb(x + y);
   // sumNums adds two numbers (x, y) and passes the result to the callback.
 }
 
+sumNums(7,10, function(sum){
+  console.log(sum)
+});
+
+
 function multiplyNums(x, y, cb) {
+  return cb(x*y);
   // multiplyNums multiplies two numbers and passes the result to the callback.
 }
 
+multiplyNums(7,7, function(product){
+  console.log(product)
+});
+
 function contains(item, list, cb) {
```

Comment:
This function isn't doing what it needs to do, the list parameter should be the array and item should be `Pencil` or one of the items in the list and checks to see if that item is found in the list